src/System.Reflection.Emit/ref/System.Reflection.Emit.cs

```diff
@@ -344,6 +344,7 @@ public sealed partial class TypeBuilder : System.Type //TYPEINFO: System.Reflect
         public override System.RuntimeTypeHandle TypeHandle { get { throw null; } }
         public override System.Type UnderlyingSystemType { get { throw null; } }
         public void AddInterfaceImplementation(System.Type interfaceType) { }
+        public System.Type CreateType() { throw null; }
```

Comment:
Is this API in netstandard? if not, then perhaps we should add a netcoreapp config instead. cc: @weshaggard